### PR TITLE
Bg2 2520 add 8.2 8.3

### PR DESCRIPTION
**Pull request by bdsl**
Adds code to start publishing version with PHP 8.2 regularly, for us to start using imminently, and also 8.3 which we might start using a little later (or potentially we could skip 8.2 and go straight to 8.3 if we find it isn't too bleeding edge).

Also removed PHP 8.0 which as mentioned on the ticket was only needed for Mautic, and we don't now expect to ever need.

### CODE REVIEW

File: php8.2-dev/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM thebiggive/php:8.2
```
Comment (bdsl) [inline php8.2-dev/Dockerfile:1]: this file is a copy of https://github.com/thebiggive/docker-php/blob/c35993238ae6f62c3e3af70045a431d506ef08d7/dev/Dockerfile , just changed 8.1 to 8.2

File: php8.2/Dockerfile [hunk 2]
```diff
@@ -1,7 +1,4 @@
-# Hopefully for a short while, we are copying the root `../Dockerfile` and
-# building otherwise the same as 8.1. Delete this as soon as stable Mautic
-# supports newer!
-FROM php:8.0-apache-buster
+FROM php:8.2-apache-bookworm
```
Comment (bdsl) [inline php8.2/Dockerfile:1]: this file is a copy of https://github.com/thebiggive/docker-php/blob/b8e9c544addee6716c42a99d6e27368a11638f8c/Dockerfile#L11-L10 , just changed 8.1-buster to 8.2-bookworm

File: php8.3-dev/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,13 @@
+FROM thebiggive/php:8.3
```
Comment (bdsl) [inline php8.3-dev/Dockerfile:1]: this file is a copy of this file is a copy of https://github.com/thebiggive/docker-php/blob/c35993238ae6f62c3e3af70045a431d506ef08d7/dev/Dockerfile , just changed 8.1 to 8.3

File: php8.3/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,33 @@
+FROM php:8.3-apache-bookworm
```
Comment (bdsl) [inline php8.3/Dockerfile:1]: this file is a copy of https://github.com/thebiggive/docker-php/blob/b8e9c544addee6716c42a99d6e27368a11638f8c/Dockerfile#L11-L10 , just changed 8.1-buster to 8.3-bookworm

File: php8.2-dev/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+# CircleCI orb requires gpg for uploader validation
+RUN apt-get update \
+ && apt-get install -y gpg \
+ && rm -rf /var/lib/apt/lists/* /var/cache/apk/*
+
+# As of April 2022 and PHP 8.1, phpdbg isn't bundled and Debian stable repos don't seem to have
+# a straightforward way to get it. So we now maintain and publish a `dev-8.1` tag with Xdebug enabled,
+# for e.g. CI test coverage. This means we can use the normal `8.1` tag for Production without
+# incurring significant extra overhead.
```
Comment (dorota-joanna) [inline php8.2-dev/Dockerfile]:
```suggestion
```

File: php8.3-dev/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+# CircleCI orb requires gpg for uploader validation
+RUN apt-get update \
+ && apt-get install -y gpg \
+ && rm -rf /var/lib/apt/lists/* /var/cache/apk/*
+
+# As of April 2022 and PHP 8.1, phpdbg isn't bundled and Debian stable repos don't seem to have
+# a straightforward way to get it. So we now maintain and publish a `dev-8.1` tag with Xdebug enabled,
+# for e.g. CI test coverage. This means we can use the normal `8.1` tag for Production without
+# incurring significant extra overhead.
```
Comment (dorota-joanna) [inline php8.3-dev/Dockerfile]:
```suggestion
```

**Review by dorota-joanna — APPROVED** [review]
LGTM

**Comment by dorota-joanna** [comment]
Thanks Dorata - Barney here merging.

**Pull request merged**